src/main/java/org/aludratest/eclipse/vde/internal/editors/VisualDataEditorPage.java

```diff
@@ -139,6 +139,17 @@ public void widgetSelected(SelectionEvent e) {
 			}
 		});
 		tiIgnore.setEnabled(false);
+		
+		final ToolItem tiExternalTestId = new ToolItem(toolbar, SWT.CHECK);
+		tiExternalTestId.setImage(VdeImage.FUNCTION.getImage());

```

Comment:
I'd appreciate using a different icon here, best would be adding a new one. Check the [Plug-In Image Browser](http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.pde.doc.user%2Fguide%2Ftools%2Fviews%2Fimage_browser_view.htm) for additional icons. I'd recommend something like a chain element (as used for links), e.g. platform:/plugin/org.eclipse.jst.jsf.standard.tagsupport/icons/palette/HTML/small/HTML_LINK.gif